.github/workflows/test.yml

```diff
@@ -51,17 +51,6 @@ jobs:
       - name: Build Dist 🔧
         run: yarn run build
 
-      - name: Build Storybook 📚
-        run: yarn workspaces foreach run build:storybook
-
-      - name: Run Accessibility Tests 💛
-        run: yarn workspaces foreach run storybook:axeOnly
```

Comment:
Seems weird to have a11y tests being part of the chromatic job (I know it's because both need the storybook build, though).

We can revisit this later, but maybe there's some renaming of things we can do to make it seem more right.